doc/getting-started/building-from-source.rst

```diff
@@ -79,8 +79,16 @@ Install the necessary tools for the following systems:
           libblkid-dev \
           e2fslibs-dev \
           pkg-config \
-          libnuma-dev
+          libnuma-dev \
+          liblz4-tool \
+          flex \
+          bison
+
      $ sudo pip3 install kconfiglib
+     $ wget https://acpica.org/sites/acpica/files/acpica-unix-20191018.tar.gz
+     $ tar zxvf acpica-unix-20191018.tar.gz
+     $ cd acpica-unix-20191018
+     $ make clean && make iasl && make install
```

Comment:
> You need `sudo` to run the `make install` step.
> 
> I also remember that we always did a `sudo cp ./generate/unix/bin/iasl /usr/sbin/` instead of a `make install` because we need the binary in `/usr/sbin` and `make install` does not put it in there. (see also https://projectacrn.github.io/latest/getting-started/rt_industry_ubuntu.html?#install-iasl-in-ubuntu-for-user-vm-launch).

Yes, I use the make install because it's the official usage, but I will change it to keep pace with the old way.